QuanLy.dart

```diff
@@ -5,7 +5,9 @@ class QuanLy{
     List<NhanVien> dsNV = [];
     QuanLy(this.dsNV);
     void inThongTin(){
```

Comment:
Please follow code convention
Suggestion: from `){` to `) {`

Reply:
I reformatted code with dart format.